wasmsdk/README.md

```diff
@@ -201,6 +246,13 @@ convert blobber urls to blobber ids
 **Output**:
 > []string
 
+### zcn.sdk.getBlobbersList
```

Comment:
do you think it is better to name it with `getBlobbers`?

Reply:
fixed in ee7cc18